segmentstore/server/src/main/java/io/pravega/segmentstore/server/reading/StreamSegmentReadIndex.java

```diff
@@ -1192,24 +1192,33 @@ private ReadResultEntryBase createFutureRead(long streamSegmentOffset, int maxLe
     }
 
     /**
-     * Gets a copy of all the ReadIndexEntries in this Index that are not RedirectReadIndices. All returned
-     * entries have their offsets adjusted by the given amount.
+     * Gets a copy of all the ReadIndexEntries in this Index that are not RedirectReadIndices and removes them from this
```

Comment:
It says "gets a copy" but then later says they are removed, so, it's not really a copy.

Reply:
Reworded.